src/_variables.scss

```diff
@@ -14,115 +10,96 @@ $clrMedium: #ccc;
-
-$btn-info-bg: $brand-info;
-$btn-info-border: darken($btn-info-bg, 20%);
-
-$btn-warning-color: #000;
-$btn-warning-bg: $brand-warning;
-$btn-warning-border: darken($btn-warning-bg, 20%);
-
-$btn-danger-bg: $brand-danger;
-$btn-danger-border: darken($btn-danger-bg, 20%);
-
-$pagination-color: $btn-default-color;
-$pagination-bg: $btn-default-bg;
-$pagination-border: $btn-default-border;
-
-$pagination-hover-color: $btn-default-color;
-$pagination-hover-bg: darken($btn-default-bg, 8%);
-$pagination-hover-border: darken($btn-default-border, 12%);
-
-$pager-border-radius: 4px;
-
-// Hack for Bootstrap 3.1.1, since it doesn't convert the "fadein" LESS method.
-// Pending fix release in https://github.com/twbs/bootstrap-sass/commit/df7be4f0412497e22f1d271d94d6e9835f61113e
-@function fadein($colour, $pct) {
-	@return opacify($colour, $pct / 100);
-}
-
-// Override Bootstrap's expected font path.
-// This is a path relative to the final CSS destination and not the current file
-$icon-font-path: "../fonts/";
-
-// Import Bootstrap variables for viewport sizes
-@import "../lib/bootstrap-sass-official/assets/stylesheets/bootstrap/variables";
+$line-height-base: 1.4375 !default;
+
+//Bootstrap colors
+$brand-primary: #2572b4 !default;
+$brand-success: #1b6c1c !default;
+$brand-info: #4d4d4d !default;
+$brand-warning: #f2d40d !default;
+$brand-danger: #bc3331 !default;
+
+$link-color: #295376 !default;
+$link-hover-color: #0535d2 !default;
+$link-visited-color: #7834bc !default;
+
+$table-bg-accent: #f5f5f5 !default;
+$table-bg-hover: #f0f0f0 !default;
+
+$btn-default-color: #335075 !default;
+$btn-default-bg: #eaebed !default;
+$btn-default-border: darken($btn-default-bg, 5%) !default;
+
+$btn-primary-bg: $brand-primary !default;
+$btn-primary-border: darken($btn-primary-bg, 20%) !default;
+
+$btn-success-bg: $brand-success !default;
+$btn-success-border: darken($btn-success-bg, 20%) !default;
+
+$btn-info-bg: $brand-info !default;
+$btn-info-border: darken($btn-info-bg, 20%) !default;
+
+$btn-warning-color: #000 !default;
+$btn-warning-bg: $brand-warning !default;
+$btn-warning-border: darken($btn-warning-bg, 20%) !default;
+
+$btn-danger-bg: $brand-danger !default;
+$btn-danger-border: darken($btn-danger-bg, 20%) !default;
+
+$pagination-color: $btn-default-color !default;
+$pagination-bg: $btn-default-bg !default;
+$pagination-border: $btn-default-border !default;
+
+$pagination-hover-color: $btn-default-color !default;
+$pagination-hover-bg: darken($btn-default-bg, 8%) !default;
+$pagination-hover-border: darken($btn-default-border, 12%) !default;
+
+$pager-border-radius: 4px !default;
+
+// Footnotes
+$spacing: .375em !default;
+$ddRtnWidth: 3.375em !default;
+
+// Calendar
+$cal-text-head: #fff !default;
+$cal-text-body: #000 !default;
+$cal-dark: #333 !default;
+$cal-medium: #555 !default;
+$cal-active: #ccc !default;
+$cal-days-link: #111 !default;
+
+//Multimedia Player
+$mm-progress-fg-color: #176ca7 !default;
```

Comment:
Not :100: sure that libsass properly supports `!default` right now, but I haven't been following the 3.x dev that close